chef_master/source/_templates/nav-docs.html

```diff
@@ -2211,84 +2177,21 @@
                 "hasSubItems": false,
                 "url": "/fips.html"
               },
-              {
-                "title": "Stream Data",
-                "hasSubItems": false,
-                "url": "/stream_data_chef_automate.html"
-              },
-              {
-                "title": "Upgrades",
-                "hasSubItems": false,
-                "url": "/upgrade_chef_automate.html"
-              }
-            ]
-          },
-          {
-            "title": "Nodes",
-            "hasSubItems": true,
-            "subItems": [
-              {
-                "title": "Node Notifications w/Slack",
-                "hasSubItems": false,
-                "url": "/integrate_node_notifications_slack.html"
-              },
-              {
-                "title": "Node Notifications w/Webhooks",
-                "hasSubItems": false,
-                "url": "/integrate_node_notifications_webhook.html"
-              },
-              {
-                "title": "Node Search Reference",
-                "hasSubItems": false,
-                "url": "/search_query_chef_automate.html"
-              }
-            ]
-          },
-          {
-            "title": "Compliance",
-            "hasSubItems": true,
-            "subItems": [
-              {
-                "title": "Reporting",
-                "hasSubItems": false,
-                "url": "/automate_compliance_reporting.html"
-              },
-              {
-                "title": "Scanner",
-                "hasSubItems": false,
-                "url": "/automate_compliance_scanner.html"
-              },
-              {
-                "title": "Profile Store",
-                "hasSubItems": false,
-                "url": "/profile_store.html"
-              },
-              {
-                "title": "Credentials",
-                "hasSubItems": false,
-                "url": "/automate_compliance_credentials.html"
-              },
-              {
-                "title": "Perform Compliance Scan",
-                "hasSubItems": false,
-                "url": "/perform_compliance_scan.html"
-              },
               {
                 "title": "Audit Cookbook",
```

Comment:
Is this something we have documented with A2? It's not part of workflow.

Reply:
Yes, but the field teams are used to finding the Audit Cookbook doc here, so I thought that I would keep it.